modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -227,6 +251,8 @@ Application:: Specifies the API to show.
 
 The Application Listing component shows applications that use an API.
 
+Data shown from Anypoint Platform: Application name, client ID, and client secret from Anypoint Exchange.
```

Comment:
This shouldn't be API Manager?

Reply:
Confirmed with eng: information in My Applications list comes from API Manager

Reply:
Updated.